pkg/client/delete.go

```diff
@@ -33,46 +35,85 @@ const (
 	e2eNamespacePrefix = "e2e-"
 )
 
+var (
+	pollFreq    = 5 * time.Second
+	pollTimeout = 5 * time.Minute
```

Comment:
Not customizable right now; will the values be a problem ever? Not sure the upper bound on namespace cleanup times. Maybe we should just bump that to something pretty big like an hour to ensure CI systems dont get hung but long enough it shouldn't get hit.